### PR TITLE
Translations incl footer

### PR DESCRIPTION
**Pull request by VirginiaDooley**
- bumps versions of the design system and dc_utils to make the header logo and footer uniform with other DC sites. 
- updates translations site wide where possible https://dev.wcivf.club/
- records where external support is needed on translations [here](https://docs.google.com/document/d/1sjfm4qwMwvkYPnZgea0M3-mMkM-NGBQEFXLVaULYryo/edit?usp=sharing)

Screenshot 2024-04-18 at 10 36 17 AM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206929626417191

### CODE REVIEW

File: wcivf/templates/base.html [hunk 1]
```diff
@@ -71,7 +71,12 @@
             </aside>
         </form>
     {% endif %}
+
 {% endblock base_language_menu %}
+{% get_current_language as LANGUAGE_CODE %}
+{% if LANGUAGE_CODE == 'cy' %}
```
Comment (VirginiaDooley) [inline wcivf/templates/base.html:77]: I can't seem to implement this the way it's suggested in the design system. The subheader site title shows in both English and Welsh versions but I only want it to appear in the latter.

**Review by pmk01 — APPROVED** [review]
Diolch!

File: wcivf/templates/base.html (hunk 1 above)
Comment (VirginiaDooley) [reply wcivf/templates/base.html:77]: @GeoWill I tried to follow what you have done for WDIV, but I'm not clear on how this works.

Comment (GeoWill) [reply wcivf/templates/base.html:77]: How is it not working? 
Have you tried printing `LANGUAGE_CODE` (or inspecting in debug toolbar) to check what it's coming out as - i.e. is it always returning `en`?

Comment (VirginiaDooley) [reply wcivf/templates/base.html:77]: The conditional isn't working, it's always on display in Welsh as shown here image

Comment (GeoWill) [reply wcivf/templates/base.html:77]: OK. So what are the results of the things I suggested?

Comment (VirginiaDooley) [reply wcivf/templates/base.html:77]: The value of `LANGUAGE_CODE` prints as `cy`

Comment (GeoWill) [reply wcivf/templates/base.html:77]: Huh. I checked it out and had a look. I think this is what you want: 
```
{% block language_code %}
    {% get_current_language as LANGUAGE_CODE %}
    {% if LANGUAGE_CODE == 'cy' %}
        <em lang="cy">Pwy alla i bleidleisio ar gyfer?</em>
    {% endif %}
{% endblock language_code %}
```
 I don't really understand why, but I guess something to do with template scopes.

Comment (GeoWill) [reply wcivf/templates/base.html:77]: There's no subtitle when it's English though. Is that what you want?

Comment (VirginiaDooley) [reply wcivf/templates/base.html:77]: I figured it out. The conditional needed to go inside the block tags. Thanks.

Comment (VirginiaDooley) [reply wcivf/templates/base.html:77]: Oh yes, you did too :)

**Pull request merged**